src/advanced_inv.cpp

```diff
@@ -861,7 +840,8 @@ void advanced_inventory_pane::add_items_from_area( advanced_inv_area &square,
             if( !cont->is_container_empty() ) {
                 // filtering does not make sense for liquid in container
                 item *it = &( square.get_container( in_vehicle() )->contents[0] );
-                advanced_inv_listitem ait( it, 0, 1, square.id );
+                // FIXME: TESTME for FALSE ???
+                advanced_inv_listitem ait( it, 0, 1, square.id, false );
```

Comment:
Uneducated guess, but why not use the value from `in_vehicle()`, as it is apparently used to get the container item in the first place? It probably doesn't even matter at all as the item (which is the contained liquid) is referenced through it's container and that is retrieved through the pane (not through the `advanced_inv_listitem`).